great_expectations/expectations/metrics/column_aggregate_metrics/column_histogram.py

```diff
@@ -214,7 +214,7 @@ def _sqlalchemy(
             )
 
         query = (
-            sa.select(case_conditions)
+            sa.select(*case_conditions)
```

Comment:
and why this `*` had to be added?

Reply:
Sure! It's because `case_conditions` is a list. The new `select()` only takes positional args, so we explode them with the `*` from a list into separate arguments.